File: app/src/main/java/mx/edu/upqroo/kristenandroid/util/KristenDateUtils.java
package mx.edu.upqroo.kristenandroid.util;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class KristenDateUtils {
    private static SimpleDateFormat mFormatter = new SimpleDateFormat("EEEE, d MMMM, yyyy",
            Locale.getDefault());

    public static String formatDate(Date date) {
        return mFormatter.format(date);
    }
}

File: app/src/main/java/mx/edu/upqroo/kristenandroid/util/KristenDateUtils.kt
package mx.edu.upqroo.kristenandroid.util

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

object KristenDateUtils {
    fun formatDate(date: Date): String {
        return SimpleDateFormat("EEEE, d MMMM, yyyy", Locale.getDefault()).format(date)
    }
}
